docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,162 @@
+= Documentation for core maintainers
+
+Jenkins core is a central component of the project which serves millions of users, and ot is critical to maintain the it in a good shape.
+We want to ensure quality of the integrated changes and continuity of the project,
+and hence the Jenkins core pull request review and merge process is more sophisticated than for the majority of plugins.
+
+== Scope of the document
+
+This document applies to the following components:
+
+* Jenkins core
+* Jenkins modules
+* Libraries included into the Jenkins core
+* Core components like Winstone, Extras Executable WAR, etc.
+
+Jenkins Remoting library is not in the scope for this document,
+because it is handled as a sub-project with independent maintainer team.
+Remoting updates in the core are subject to the process though.
+
+== Team
+
+=== Communication channels
+
+* link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer Mailing List]
+* Chat: N/A, to be created
+* Office hours: N/A, to be created
+
+=== Maintainer roles
+
+* Committer / reviewer
+* Core Pull Request Reviewer
+* Core Maintainer
+* Release Team Member
+
+**Committers** and **reviewers** are common contributors who submit their changes and who review changes/issues submitted by others.
+There is no special preconditions to do so, anyone is welcome to contribute.
+
+**Core Pull Request Reviewers** is the next step for contributors who are interested to review Jenkins pull requests on a regular basis.
+They get [Triage permissions](TODO) so that they can manage pull requests, request reviews and prepare changelog drafts in the pull request description.
```

Comment:
Perhaps this would make sense what exactly triage permissions is
```suggestion
They get [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization) so that they can manage pull requests, request reviews and prepare changelog drafts in the pull request description.
```